projects/bitcoin-core/project.yaml

```diff
@@ -0,0 +1,20 @@
+homepage: "https://github.com/bitcoin/bitcoin"
+main_repo: 'https://github.com/bitcoin/bitcoin.git'
+language: c++
+primary_contact: "marco@chaincode.com"
```

Comment:
Is this correct?

Reply:
It is the google account I signed up with (for oss-fuzz). If you wanted to disable the email, that is fine, too. Then I'll create a fresh account.